js/components/main.js

```diff
@@ -424,16 +445,20 @@ class Main extends ImmutableComponent {
 
   onDragOver (e) {
     let intersection = e.dataTransfer.types.filter((x) => ['Files'].includes(x))
-    if (intersection.length > 0) {
+    if (intersection.length > 0 || e.dataTransfer.getData('text/plain')) {
       e.dataTransfer.dropEffect = 'copy'
       e.preventDefault()
     }
   }
 
   onDrop (e) {
-    if (e.dataTransfer.files) {
+    if (e.dataTransfer.files.length > 0) {
       Array.from(e.dataTransfer.files).forEach((file) =>
         windowActions.newFrame({location: file.path, title: file.name}))
+    } else if (e.dataTransfer.getData('text/plain')) {
+      windowActions.setNavBarUserInput(e.dataTransfer.getData('text/plain'))
```

Comment:
right now this just loads the url into the urlbar. Do we want it to actually load the url?